index.html

```diff
@@ -484,6 +490,115 @@
               layout="[[layoutClass]]"
               route="{{subRoute}}"></generic-page>
           </template>
+          <!-- LURES    ============================================   -->
```

Comment:
> Yes it will include a different info arch and will need sub-pages which the current site doesn't allow for.

You may have misinterpreted what I said. The current top-level pages are board-agnostic; they are built to support board specific sub-pages. Eg., https://minnowboard.org/board-viewer/minnowboard_turbot_B_41/JP1Pin15

In the above, if the minnowboard3 was populated, you could replace `minnowboard_turbot_B_41/JP1Pin15` with `minnowboard3` and have the v3 board load.

The same pattern is intended to apply to `where-to-buy`, `learn-more`, etc. allowing there to be a content-type-specific sub-page for the categories already there.

The concerns I wanted to raise were:

1) Creating top-level URLs that you know going to move around are going to create 404s. If you know what the URL is going to be long term, set up the site to use those URLs now. If you don't expect people to create any links to those pages, then 404s are not a worry and carry on.

2) There are easier ways to implement and support the lures than creating N-unique top level pages.

I was offering to help set up the above. Sounds like you have a plan, so I'll let you run with it.

James